dev-packages/e2e-tests/test-applications/create-remix-app-express/tests/behaviour-server.test.ts

```diff
@@ -0,0 +1,41 @@
+import { expect, test } from '@playwright/test';
+import { waitForError, waitForTransaction } from 'event-proxy-server';
+
+test('Sends a loader error to Sentry', async ({ page }) => {
+  const loaderErrorPromise = waitForError('create-remix-app-express', errorEvent => {
+    return errorEvent.exception.values[0].value === 'Loader Error';
+  });
+
+  await page.goto('/loader-error');
+
+  const loaderError = await loaderErrorPromise;
+
+  expect(loaderError).toBeDefined();
+  expect(loaderError.contexts.trace.op).toBe('http.server');
+});
+
+test('Sends form data with action error to Sentry', async ({ page }) => {
+  await page.goto('/action-formdata');
+
+  await page.fill('input[name=test]', 'test');
+  await page.setInputFiles('input[type=file]', {
+    name: 'file.txt',
+    mimeType: 'text/plain',
+    buffer: Buffer.from('this is test'),
+  });
+
+  const formdataActionTransaction = waitForTransaction('create-remix-app-express', transactionEvent => {
+    return transactionEvent?.spans?.some(span => span.op === 'function.remix.action');
+  });
+
+  await page.click('button[type=submit]');
+
+  const actionTransaction = await formdataActionTransaction;
+
+  expect(actionTransaction).toBeDefined();
+  expect(actionTransaction.contexts.trace.op).toBe('http.server');
+  expect(actionTransaction.spans[0].data).toMatchObject({
+    action_form_data_test: 'test',
+    action_form_data_file: 'file.txt',
+  });
+});
```

Comment:
Could you also add the test I added here (or a similar one):

https://github.com/getsentry/sentry-javascript/pull/11077/files#diff-a429d23d342f5ef19e60ebce01d9d6bd7d678a2f11564a4e8a8833723fc6c08e